eth/api.go

```diff
@@ -1187,6 +1187,13 @@ func (s *PublicTransactionPoolAPI) GetTransactionReceipt(txHash common.Hash) (ma
 		if err != nil {
 			return nil, err
 		}
+
+		if err := core.WriteReceipts(s.chainDb, receipts); err != nil {
+			glog.V(logger.Warn).Infof("cannot save updated receipts: %v", err)
```

Comment:
Do you think `Warn/Info` is an appropriate level/severity for this? Or even possibly critical failure/error?